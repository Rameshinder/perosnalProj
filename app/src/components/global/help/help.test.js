import React from 'react'
import {shallow} from 'enzyme'

import Help from './index'

jest.mock('@anz/icon/filled/arrows-and-symbols/question-help', () => 'HelpIcon')

describe('Help', () => {
  const app = shallow(<Help />)

  test('renders properly', () => {
    expect(app).toMatchSnapshot()
  })

  it('Help should render without crashing', () => {
    shallow(<Help />)
  })
})
