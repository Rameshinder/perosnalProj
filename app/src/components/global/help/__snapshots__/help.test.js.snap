// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Help renders properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Help />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": Array [
        <Icon
          iconName="Help"
        />,
        <span>
          Help
        </span>,
      ],
      "href": undefined,
      "tabIndex": 0,
      "target": "_blank",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "iconName": "Help",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Help",
        },
        "ref": null,
        "rendered": "Help",
        "type": "span",
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <Icon
            iconName="Help"
          />,
          <span>
            Help
          </span>,
        ],
        "href": undefined,
        "tabIndex": 0,
        "target": "_blank",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "iconName": "Help",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Help",
          },
          "ref": null,
          "rendered": "Help",
          "type": "span",
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
